File: javascript/empleados/Comercial.js
const Empleado = require('./Empleado');

module.exports = class Comercial extends Empleado{

    zona;
    constructor(nombre,edad,salario,comision) {
        super(nombre,edad);
        if(edad < 25 && comision > 200)
            this.salario = salario + this.PLUS + comision;
        else
        this.salario = salario + comision;
        this.comision = comision;
    }

    getComercial(){
        return "Nombre: " + this.nombre +
        "\nEdad: " + this.edad + 
        "\nSalario: " + this.salario + 
        "\nComision: " + this.comision ;
    }

}

File: python/Empleados/Comercial.py
from Empleados import Empleados

class Comercial(Empleados):

    __comision = 0


    def __init__(self, nombre, edad, salario,comision):
        salario = salario + comision
        if edad>30 and comision > 200:
            salario = salario + super().PLUS
        super().__init__(nombre, edad, salario)
        self.__comision = comision
    
    def __str__(self):
        return super().__str__() + "\nComision: " + str(self.__comision)